media-sound/supercollider/supercollider-3.8.0.ebuild

```diff
@@ -35,6 +35,7 @@ RDEPEND="
 	)
 	sndfile? ( media-libs/libsndfile )
 	wiimote? ( app-misc/cwiid )
+	server? ( !app-admin/supernova )
```

Comment:
Next time, please order the flag conditionals lexically. I'm going to fix it for you now.